modules/cron/src/Controller/Cron.php

```diff
@@ -115,13 +115,7 @@ public function info(): Template
     public function run(string $tag, string $key, string $output = 'xhtml'): Response
     {
         $configKey = $this->cronconfig->getOptionalString('key', 'secret');
```

Comment:
This is rather implicit. If you forget or misset the key, it will be "secret". Maybe just make it a nonoptional string which would give a much more concrete result?